pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -262,7 +262,7 @@ stdenv.mkDerivation rec {
     description = "A cross-language development platform for in-memory data";
     homepage = "https://arrow.apache.org/docs/cpp/";
     license = licenses.asl20;
-    broken = (stdenv.isLinux && stdenv.isAarch64) || stdenv.isDarwin; # waiting on gtest changes in staging
+    broken = stdenv.isLinux && stdenv.isAarch64; # waiting on gtest changes in staging
```

Comment:
Is the comment still accurate?

Reply:
Yes, waiting on the next staging-next iteration (TBD).